packages/design-system/src/components/swatch/stories/index.js

```diff
@@ -151,16 +151,8 @@ const VARIANTS = [
     Icon: Cross,
   },
   {
-    variant: 'Large, pipette',
+    variant: 'Large, pipette, disabled',
```

Comment:
Nit: Since we don't have the small variant separately anymore, maybe remove the references to "Large" as well?

Reply:
Updated texts in 96a41bcce7.